docs/python/settings-reference.md

```diff
@@ -52,18 +52,22 @@ The language server settings apply when `python.languageServer` is `Pylance` or
+| ignore | [] | Paths of directories or files which diagnostic output (errors and warnings) should be suppressed, even if they are an included file or within the transitive closure of an included file. Paths may contain wildcard characters such as `**` (a directory or multiple levels of directories), `*` (a sequence of zero or more characters), or `?` (a single character). If no value is provided, the value of `python.linting.ignorePatterns` (if set) will be used. |
 | stubPath | ./typings | Specifies a path to a directory that contains custom type stubs. Each package's type stub file(s) are expected to be in its own subdirectory. |
 | autoSearchPaths | true | Indicates whether to automatically add search paths based on some predefined names (like `src`). Available values are `true` and `false`. |
-| extraPaths | [] | Specifies extra search paths for import resolution. Paths should be specified as strings and must be separated by commas when there are multiple paths. `["path 1","path 2"]` |
+| extraPaths | [] | Specifies extra search paths for import resolution. Accepts paths specified as strings and separated by commas if there are multiple paths. For example: `["path 1","path 2"]` |
+| indexing | true | Used to specify whether Pylance should index user files as well as installed third party libraries at start up, to provide a more complete set of symbols in features such as auto-imports, quick fixes, auto completions, etc. Accepted values are `true` or `false`. When set to `true`, by default Pylance indexes top-level symbols of installed packages (i.e., symbols in `__all__` under `package/__init__.py`), as well as all symbols from up to 2000 user files. When set to `false`, Pylance will only display symbols already referenced or used in files that were previously opened in the editor.    |
+| packageIndexDepths | [] | Used to override how many levels under installed packages to index on a per package basis. By default, only top-level modules are indexed (depth = 1). To index submodules, increase depth by 1 for each level of submodule you want to index. Accepts values are tuples of [```package name```, ```depth to scan```, ```include all symbols?```]. If `include all symbols?` is set to `false`, only symbols in each package's `__all__` are included. When it's set to `true`, Pylance will index every module/top level symbol declarations in the file. Usage example: ```[["pandas", 3, true], ["numpy", 2, false]] ``` |
```

Comment:
```suggestion
| packageIndexDepths | [] | Used to override how many levels under installed packages to index on a per package basis. By default, only top-level modules are indexed (depth = 1). To index submodules, increase depth by 1 for each level of submodule you want to index. Accepted values are tuples of [```package name```, ```depth to scan```, ```include all symbols?```]. If `include all symbols?` is set to `false`, only symbols in each package's `__all__` are included. When it's set to `true`, Pylance will index every module/top level symbol declarations in the file. Usage example: ```[["pandas", 3, true], ["numpy", 2, false]] ``` |
```